src/components/GuidelineCard/GuidelineCard.module.css

```diff
@@ -29,6 +29,18 @@
   margin-bottom: 32px;
 }
 
+.details {
+  margin-bottom: 32px;
+}
+
+.criterion h4 {
+  margin-bottom: 4px;
+}
+
+.criterion:not(:last-child) {
+  margin-bottom: 32px;
+}
```

Comment:
Question: Is it normal to define margins with px instead of percentage or points? :)

Reply:
I'd say yes, unless you want a dynamic margin like "5% of container width". I think case think we want them pretty fixed.